typha/fv-tests/server_test.go

```diff
@@ -1611,6 +1612,19 @@ var _ = Describe("with server requiring TLS", func() {
 		}
 	}
 
+	testTcpHalfOpen := func() {
+		serverAddr := fmt.Sprintf("127.0.0.1:%d", server.Port())
+		tcpConn, err := net.Dial("tcp", serverAddr)
+		Expect(err).NotTo(HaveOccurred())
+		err = tcpConn.SetDeadline(time.Now().Add(30 * time.Second))
+		Expect(err).NotTo(HaveOccurred())
+		deadline := time.Now().Add(10 * time.Second)
```

Comment:
This calculation happens in parallel with the server's calculation so it races with that; suggest moving this line to before the `net.Dial()` and maybe reducing to 9s, just to allow some leeway.

Reply:
sure thing, thanks for the suggestion, it has been applied, could you please have another look @fasaxc ?